src/Redux/slices/userSlice.js

```diff
@@ -0,0 +1,32 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+export const consultUser = createAsyncThunk('get/consultUser', async (name) => {
+  try {
+    const response = await axios.get(`http://localhost:3000/api/v1/users/${name}`);
+    return response.data.exist;
+  } catch (error) {
+    return Promise.reject(error);
+  }
+});
+
+const initialState = {
+  user: null,
+  userName: null,
+};
+
+const userSlice = createSlice({
```

Comment:
Kindly remove this file. I can notice your effort and hard work on this implementation as it is needed to create and test the login page, however this `UserSlice` corresponds to another task, please don't get ahead with the tasks.

Reply:
Tky for your time and your comments, Solved